test/modules/aws-acm-import-integration.ts

```diff
@@ -148,7 +148,7 @@ describe('AwsAcm Import Integration Testing', () => {
   itDocs(
     'import a certificate in non-default region',
     query(`
-    SELECT * FROM certificate_import('${cert}', '${key}', '${region}', '{}');
+    SELECT * FROM certificate_import('${cert}', '${key}', 'us-east-1', '{}');
```

Comment:
Why this change?

Reply:
It was using the default region. It should be non-default region instead.